src/MercadoLibre/Provider.php

```diff
@@ -57,9 +57,9 @@ public static function additionalConfigKeys()
      */
     protected function getAuthUrl($state)
     {
-        $url = self::DOMAIN[$this->getConfig('country')] ?? 'https://auth.mercadolibre.com.ar';
```

Comment:
I actually think the original is better, as if they pass an invalid country code it won't cause an error. What do you think?